src/vm/frames.h

```diff
@@ -2519,18 +2519,30 @@ class GCFrame : public Frame
     }
 #endif
 
-#if defined(_DEBUG_IMPL)
-    const char* GetFrameTypeName() { LIMITED_METHOD_CONTRACT; return "GCFrame"; }
-#endif
+    PTR_GSCookie GetGSCookiePtr()
```

Comment:
We do not need any of this GSCookie stuff for this.

Reply:
The manual GSCookies were designed for defense in depth of frames in JITed and assembly code. GCFrames are always in C/C++ code. It is fine to depend on the defense in depth provided by the C/C++ compiler for these.